graftm/hmmer.py

```diff
@@ -31,6 +31,13 @@ def __init__(self, search_hmm, aln_hmm=None):
         self.search_hmm = search_hmm
         self.aln_hmm = aln_hmm
 
+    def _get_sequence_directions(self, search_result):
+        complement_strand = {}
```

Comment:
probably shouldn't use the word complement, I know wht you mean but others likely won't. To be technical, the complement of a forward read AG is TC, not 'forward'